content/hi/docs/reference/glossary/kubectl.md

```diff
@@ -0,0 +1,18 @@
+---
+title: kubectl
+id: kubectl
+date: 2018-04-12
+full_link: /docs/user-guide/kubectl-overview/
+short_description: >
+  कुबेरनेट्स क्लस्टर के साथ संचार करने के लिए एक कमांड लाइन उपकरण।
+aka:
+- kubectl
+tags:
+- tool
+- fundamental 
+---
+कुबेरनेट्स API का उपयोग करके कुबेरनेट्स क्लस्टर के {{< glossary_tooltip text="कण्ट्रोल प्लेन" term_id="control-plane">}} के साथ संचार करने के लिए कमांड लाइन उपकरण।
+
+<!--more-->
+
+आप कुबेरनेट्स ऑब्जेक्ट बनाने, निरीक्षण करने, अपडेट करने और हटाने के लिए `kubectl` का उपयोग कर सकते हैं।
```

Comment:
```suggestion
आप कुबेरनेट्स ऑब्जेक्ट को बनाने, निरीक्षण करने, अपडेट करने और हटाने के लिए `kubectl` का उपयोग कर सकते हैं।
```

Nit

Reply:
@sachinparihar Feel free to commit this one!